types/rr-notifications/index.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for rr-notifications 1.1
+// Project: https://thegrinder.github.io/rr-notifications
+// Definitions by: Robbie Moore <https://github.com/DefinitelyTyped>
```

Comment:
Should be your username here

Reply:
Got it, changed `DefinitelyTyped` to `RobbieGM`. I assume you meant to change that rather than changing `Robbie Moore` to `RobbieGM` right?